source/features/jump-to-conversation-close-event.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import {observe} from 'selector-observer';
+import * as pageDetect from 'github-url-detection';
+
+import {wrap} from '../helpers/dom-utils';
+import features from '.';
+
+function init(): Deinit {
+	return observe('#partial-discussion-header :is([title="Status: Closed"], [title="Status: Merged"], [title="Status: Closed as not planned"]):not(.rgh-jump-to-conversation-close-event)', {
+		constructor: HTMLSpanElement,
```

Comment:
This shouldn't be necessary

Reply:
I  am changing the style which will give an error on type element

Reply:
`HTMLElement`